raft/src/main/java/org/apache/kafka/raft/QuorumConfig.java

```diff
@@ -69,23 +70,24 @@ public class QuorumConfig {
 
     public static final String QUORUM_ELECTION_TIMEOUT_MS_CONFIG = QUORUM_PREFIX + "election.timeout.ms";
     public static final String QUORUM_ELECTION_TIMEOUT_MS_DOC = "Maximum time in milliseconds to wait " +
-        "without being able to fetch from the leader before triggering a new election";
+        "without being able to fetch from the leader before triggering a new election, this value should large 0.";
     public static final int DEFAULT_QUORUM_ELECTION_TIMEOUT_MS = 1_000;
 
     public static final String QUORUM_FETCH_TIMEOUT_MS_CONFIG = QUORUM_PREFIX + "fetch.timeout.ms";
     public static final String QUORUM_FETCH_TIMEOUT_MS_DOC = "Maximum time without a successful fetch from " +
         "the current leader before becoming a candidate and triggering an election for voters; Maximum time " +
-        "a leader can go without receiving valid fetch or fetchSnapshot request from a majority of the quorum before resigning.";
+        "a leader can go without receiving valid fetch or fetchSnapshot request from a majority of the quorum before resigning, " +
+        "this value should large 0.";
```

Comment:
ditto: I don't think we need to add this to the document; the valid value section already describes this limitation.